autospec/tarball.py

```diff
@@ -451,10 +471,10 @@ def process(url_arg, name_arg, ver_arg, target, archives_arg, filemanager):
     version = ver_arg
     archives = archives_arg
     tarfile = os.path.basename(url_arg)
-    # set gcov file information
-    set_gcov()
     # determine name and version of package
     name_and_version(name_arg, ver_arg, filemanager)
+    # set gcov file information, must be done after name is set
```

Comment:
This isn't immediately obvious to me for why, could be added in the comment here (or in the commit message but here seems easier).

Reply:
Comment updated. gcov_file variable is set to "<tarball.name>.gcov" if the gcof file exists.

Reply:
Ah okay, the gcov stuff, thanks!